modules/test/playwright/tests/data-set-manager/actions.spec.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {dataSetManagerPagesTest} from '../../fixtures/dataSetManagerPageTest';
+import {loginTest} from '../../fixtures/loginTest';
+
+export const test = mergeTests(dataSetManagerPagesTest, loginTest);
+
+test('There are no item actions created in the Action tab', async ({
+	dataSetManagerActionsPage,
+	dataSetManagerViewsPage,
+	page,
+}) => {
+	await dataSetManagerViewsPage.goto();
+	await dataSetManagerActionsPage.goto();
+
+	await expect(
+		page.getByRole('tabpanel').getByText('No actions were created.')
+	).toBeVisible();
+});
+
+test('The "New Creation Action" button is present', async ({
+	dataSetManagerActionsPage,
+	dataSetManagerViewsPage,
+	page,
+}) => {
+	await dataSetManagerViewsPage.goto();
+	await dataSetManagerActionsPage.goto();
+
+	await expect(
+		page.getByRole('button', {name: 'New Item Action'})
+	).toBeVisible();
+});
+
+test('A new Link action is created', async ({
+	dataSetManagerActionsPage,
+	dataSetManagerViewsPage,
+	page,
+}) => {
+	await dataSetManagerViewsPage.goto();
+	await dataSetManagerActionsPage.goto();
+
+	await dataSetManagerActionsPage.createTestDataSetAction({
+		icon: 'arrow-right-full',
+		name: 'Link action',
+		type: 'link',
+		url: 'http://localhost:8080',
```

Comment:
yes, there's a liferayConfig for it -> https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/liferay.config.ts

And you use it like this -> https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/fixtures/loginTest.ts#L8

Reply:
In this case, I think making the url an argument makes the test more flexible, since we'll could test over different url flavors (with parameters inside or not, against certain APIs, and so on).

Reply:
That's OK, but in that case, don't pass `http://localhost:8080` as the url argument, instead pass `liferayConfig.environment.baseUrl` because otherwise your test may fail in the CI.